extensions/arc/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+	"extends": "../shared.tsconfig.json",
+	"compileOnSave": true,
+	"compilerOptions": {
+		"outDir": "./out",
+		"lib": [
+			"es6",
+			"es2015.promise"
+		],
+		"strict": false
```

Comment:
It'd be great if you could not disable this and then just fix up the issue it finds. Or is the swagger api causing too many issues?

Reply:
I will turn strict on.  The swagger clients are fine actually.  My extension code caused 2 issues:

(1) 'string | undefined' is not assignable to type 'string'
(2) Object is possibly 'undefined

Which I can fix with ! and null checks.